modules/react-components/src/components/input/copy-input-field.tsx

```diff
@@ -55,9 +55,9 @@ export interface CopyInputFieldPropsInterface extends IdentifiableComponentInter
 /**
  * Copy to clipboard input field component.
  *
- * @param {CopyInputFieldPropsInterface} props - Props injected to the component.
+ * @param props - Props injected to the component.
  *
```

Comment:
Can we remove the redundant line?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3477/commits/33110800011c556fb355e88506b69eade2e95cef